kapitan/helm_cli.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import subprocess
+from subprocess import PIPE, DEVNULL
+
+logger = logging.getLogger(__name__)
+
+
+def helm_cli(args, stdout=None, verbose=False):
+    try:
+        logger.debug("launching helm with arguments: %s", args)
+        res = subprocess.run(
+            args=["helm"] + args, stderr=PIPE, stdout=stdout or (PIPE if verbose else DEVNULL)
```

Comment:
I think it would make sense if we can somehow configure which helm binary to use instead of relying on a hardcoded value. Some users might have a dual-helm setup with helm v2 and v3 and their helm binary might be called `helm3` or so. Maybe try to use the binary in an env var like "HELM_BIN" (or whatever appropriate) and default to `helm` if not defined?

e.g. https://github.com/roboll/helmfile allows to set the "helmBinary" config.

Reply:
For that, I think it would be best to support a `helm_path` option along the other ones

Reply:
Both use cases are valid IMHO. Some people may need to set the path globally (if they want to use a specific helm for everything), and some will need to set it per compile action (to compile a `helm2` chart for instance).

Reply:
yeah, sure. being able to specify it globally or per compilation should give enough flexibility